### ISSUE TITLE
non-https View buttons are no longer red

### ISSUE THREAD

**Issue by DandelionSprout** [collinbarrett/FilterLists#499]
I can't find the original issue that was about giving `View`/`Subscribe` HTTP links red buttons, but I just wanted to point out that for lists with multiple view/subscription links, not all HTTP or insecure-SSL buttons are red anymore.

Example: MVPS Hosts → `Details` → `<View`, and the `Original` button is still blue. There's probably some other examples of this as well.

![image](https://user-images.githubusercontent.com/22780683/45827712-5394ba00-bcf7-11e8-9d52-3028428fe03e.png)

**Comment by collinbarrett** [comment]
So, I recently did a major refactor of the UI. This project is the first and only I've done in React, so I had initially built two monolith components (the main Home table, and the details expanders). I chopped them up into a bunch of tiny components to be more flexible and reusable.

Anyway...

I realized when doing so that it didn't make much sense to have the `View` buttons change to red for non-`https` and not all of the other buttons (`Home`, `Donate`, etc...). The common UX on the web is that hyperlinks don't look any different visually whether they are `http` of `https`. If one wants to know before clicking, they can typically hover over the link URL in the bottom corner of most browsers.

So, my thinking was keep the functionality for the `Subscribe` buttons, as that's a different paradigm than most hyperlinks. It helps users spot right away that they are subscribing to an unencrypted source. But, I thought for all the other links, including the `View` buttons, we should stick with no differentiation to match the common UX of the web.

But, I am open to reconsider. I just think we should be consistent, and making all of the buttons dynamic could result in a very colorful/confusing UI.
